core-concepts/wallets/veworld/user-guide/activities.md

```diff
@@ -0,0 +1,61 @@
+---
+description: How to access your history
+layout:
+  title:
+    visible: true
+  description:
+    visible: true
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# Activities
+
+{% tabs %}
+
+{% tab title="Mobile" %} 
+
+To watch your history you should:
+
+1. Go to your Dashboard
+2. Tap the History button
+3. You'll see a list of activies (some of them are onchain, some aren't)
+4. Tap one of them to read more
+5. In each details you'll see:
+    - From and To Address
+    - Datetime of creation
+    - Value (which and how many tokens)
+    - Transaction ID (you can also copy it in your clipboard)
+    - Blocknumber
+    - Network
+    - Button to read details of that transaction in the Blockexplorer
+
+{% endtab %}
+
+{% tab title="Browser Extension" %}
+
+To watch your history you should:
+
+1. Go to your Dashboard
+2. Click the History button
+3. You'll see a list of activies (some of them are onchain, some aren't)
+4. Click one of them to read more
+5. In each details you'll see:
+    - Activity type
+    - Status
+    - Transaction ID (you can also copy it in your clipboard)
+    - Network
+    - Finality
+    - Datetime of creation
+    - From and To Address
+    - Value (which and how many tokens)
+    - Button to read details of that transaction in the Blockexplorer
+
+
```

Comment:
Any reason why you are not using screenshots here?

Reply:
There's an ongoing conversation with the IT Team about making some screecasts/tutorials.
I'm waiting for their investigations.

Reply:
@victhorbi